openmc/cross.py

```diff
@@ -309,7 +309,7 @@ def __deepcopy__(self, memo):
             clone._right_filter = self.right_filter
             clone._binary_op = self.binary_op
             clone._type = self.type
-            clone._bins = self.bins
+            clone._bins = self._bins
```

Comment:
Since `self.bins` returns the tuple `self._bins['left'], self._bins['right']`, the value of `_bins` for the cloned CrossFilter will be a tuple, and not the expected dictionary.